static/js/tippyjs.js

```diff
@@ -158,4 +161,33 @@ export function initialize() {
         hideOnClick: true,
         theme: "light-border",
     });
+
+    // compose box buttons popover shown on mobile widths.
+    delegate("body", {
```

Comment:
I feel really nice about having all the code related to a popover in a single code block.

Reply:
Yeah, this feels pretty readable.